### PR TITLE
feat: TorchObjectDetectionSegmenter

### PR DESCRIPTION
**Pull request by cristianmtr**
AFTER MERGING

- update old README
- archive old repo

@cristianmtr

### CODE REVIEW

File: jinahub/crafters/TorchObjectDetectionSegmenter/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+To install the dependencies locally run 
+```
+pip install . 
+```
+
+To verify the installation works:
+```
+pip install -r tests/requirements.txt
+pytest -sv tests
+```
```
Comment (mapleeit) [inline jinahub/crafters/TorchObjectDetectionSegmenter/README.md:27]: We do not recommend user to use it in this way, right?

File: jinahub/crafters/TorchObjectDetectionSegmenter/manifest.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+manifest_version: 1
+name: torch_object_detection_segmenter
+alias: TorchObjectDetectionSegmenter
+description: Torch Object Detection Segmenter
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://github.com/jina-ai/EXECUTOR_REPO_NAME
```
Comment (mapleeit) [inline jinahub/crafters/TorchObjectDetectionSegmenter/manifest.yml:6]:
```suggestion
url: https://github.com/jina-ai/executors/jinahub/crafters/TorchObjectDetectionSegmenter
```

File: jinahub/crafters/TorchObjectDetectionSegmenter/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+torch==1.8.1
+torchvision==0.9.1
+jina~=2.0
```
Comment (mapleeit) [inline jinahub/crafters/TorchObjectDetectionSegmenter/requirements.txt:3]:
```suggestion

```

**Review by mapleeit — CHANGES_REQUESTED** [review]

**Review by Yongxuanzhang — COMMENTED** [review]
Looks like this is a `segmenter`, not a `crafter`?

**Comment by cristianmtr** [comment]
> Looks like this is a `segmenter`, not a `crafter`?

Do we have this differentation? I thought they would both be under `crafters`

File: jinahub/crafters/TorchObjectDetectionSegmenter/README.md (hunk 1 above)
Comment (cristianmtr) [reply jinahub/crafters/TorchObjectDetectionSegmenter/README.md]: yep, good catch. fixed

File: jinahub/crafters/TorchObjectDetectionSegmenter/manifest.yml (hunk 2 above)
Comment (cristianmtr) [reply jinahub/crafters/TorchObjectDetectionSegmenter/manifest.yml]: fixed

File: jinahub/crafters/TorchObjectDetectionSegmenter/requirements.txt (hunk 3 above)
Comment (cristianmtr) [reply jinahub/crafters/TorchObjectDetectionSegmenter/requirements.txt]: removed

**Comment by Yongxuanzhang** [comment]
> > Looks like this is a `segmenter`, not a `crafter`?
> 
> Do we have this differentiation? I thought they would both be under `crafters`

segmenters are used to create chunks, crafters are used to modify data, to my understanding.

**Comment by cristianmtr** [comment]
> > > Looks like this is a `segmenter`, not a `crafter`?
> > 
> > 
> > Do we have this differentiation? I thought they would both be under `crafters`
> 
> segmenters are used to create chunks, crafters are used to modify data, to my understanding.

right, I'll split them

**Review by Yongxuanzhang — APPROVED** [review]

**Pull request merged**